docker-compose.yml

```diff
@@ -2,60 +2,61 @@
 version: "3.8"
 
 services:
+  # runs if started with `task env-up DB=ferretdb`
   ferretdb:
     image: ${FERRETDB_IMAGE:-ferretdb-local}
     container_name: dance_ferretdb
     restart: unless-stopped
-    depends_on: ["postgres", "mongosh"]
+    depends_on: [ "postgres", "mongosh" ]
     ports:
-      - 127.0.0.1:27017:27017
-    command:
-      [
-        "-test.coverprofile=cover.txt",
-        "--",
-        "--listen-addr=:27017",
-        "--postgresql-url=postgres://user@postgres:5432/dance",
-        "--mode=normal",
-      ]
+      - 27017:27017
+    command: -test.coverprofile=cover.txt --
     environment:
       # Always UTC+05:45. Set to catch timezone problems.
       - TZ=Asia/Kathmandu
+      - FERRETDB_POSTGRESQL_URL=postgres://user@postgres:5432/dance
 
+  # runs if ferretdb also runs
   postgres:
     build:
       context: ./build/deps
       dockerfile: postgres.Dockerfile
     container_name: dance_postgres
     restart: unless-stopped
     ports:
-      - 127.0.0.1:5432:5432
+      - 5432:5432
     environment:
       # UTC−03:30/−02:30. Set to catch timezone problems.
       - TZ=America/St_Johns
       - POSTGRES_USER=user
       - POSTGRES_DB=dance
       - POSTGRES_HOST_AUTH_METHOD=trust
 
+  # runs if started with `task env-up DB=mongodb`
   mongodb:
     build:
       context: ./build/deps
       dockerfile: mongo6.Dockerfile
     command: --setParameter enableTestCommands=1
     container_name: dance_mongodb
     restart: unless-stopped
-    depends_on: ["mongosh"]
+    depends_on: [ "mongosh" ]
     ports:
-      - 127.0.0.1:27017:27017
+      - 27017:27017
     environment:
       # Always UTC+05:45. Set to catch timezone problems.
       - TZ=Asia/Kathmandu
 
+  # for tools; always runs, but no ports are exposed
   mongosh:
     build:
       context: ./build/deps
       dockerfile: mongo6.Dockerfile
+    command: --port 12345 # should not be used
```

Comment:
I don't understand this line
What's the point of specifing the port and what do you mean by "should not be used"?

Reply:
Oh okay, I guess it's set to remove collidation on 27017 port